pilot/pkg/networking/core/v1alpha3/listener_builder.go

```diff
@@ -141,12 +141,9 @@ func (builder *ListenerBuilder) buildVirtualListener(
 	return builder
 }
 
+// TProxy uses only the virtual outbound listener on 15001 for both directions
```

Comment:
Oh, I didn't know that TProxy is used for only outbound. 
What do you mean by "for both directions"?

Reply:
Per the iptables code, Tproxy seems to be using only the standard virtual listener and not the virtual inbound listener. thats what this comment means.